.vscode/launch.json

```diff
@@ -4,23 +4,45 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "shopify app build",
+        "cwd": "${workspaceFolder}",
+        "runtimeExecutable": "pnpm",
+        "runtimeArgs": [
+          "shopify",
+          "app",
+          "build",
+          "--path=${input:appDir}",
+        ],
+        "env": {
+          "DEBUG": "\"*\"",
+          "SHOPIFY_DEBUG_GO_BINARY": "1",
+        },
+        "skipFiles": [
+          "<node_internals>/**",
+        ],
+      },
       {
         "type": "node",
         "request": "launch",
         "name": "shopify app dev",
-        "runtimeExecutable": "yarn",
+        "cwd": "${workspaceFolder}",
+        "runtimeExecutable": "pnpm",
```

Comment:
Is there a reason why we replaced the `yarn` executable? We did some experimentation with `pnpm` but we haven't made a decision yet on whether we should revisit our default option.